### DISCUSSION TITLE
Collaborative Development is still under development

### DISCUSSION THREAD

**Discussion by kinetikGitHub** [joyfullservice/msaccess-vcs-addin#345] [General]
This page https://github.com/joyfullservice/msaccess-vcs-integration/wiki/Documentation has this phrase:
_Note: This aspect of building the database from source files is still under development._
It has had this phrase at least since Nov-2020.
When is this going to be lifted? I take it to mean this is a beta feature and should not be used for Production.

If I used it anyway, how would I become aware of a problem? I can recognize a crash of course, but what about a subtle bug in a little used part of my large app?

**Comment by joyfullservice** [comment]
Hi Tom, thanks for reaching out! That page could probably use some updating...  😄  The intended meaning is that the add-in has not been extensively tested in a wide range of environments, and may have some potential issues in certain edge-case scenarios. If it's any encouragement, I have been using the system in production for years at my workplace and find it to meet our needs quite nicely.

The disclaimer is there because the idea of building a database completely from source files, including some of the additional types of content exported by this add-in is something outside of the normal supported usage of Microsoft Access. That being said, the ability to build from source is a pretty revolutionary concept that has transformed how we do development where I work.

Whether you are comfortable using it in production is really going to be up to you. Personally, I am very comfortable using it with our in-house applications that we use for mission-critical systems and processes, but that comfort level has come over time and experience with the add-in and our internal development workflow. In my opinion, I actually feel _safer_ using the add-in because I know that at any point I can simply rebuild a project from source, going back to any version I need.

As to your question of how to identify any issues with your large app, probably the best way is to simply build it from source, then put it through its paces and make sure everything looks and works like it is supposed to. Some things to watch for would be layout/styling issues (in rare cases with legacy projects that have been upgraded from very old versions of Access) or if you have any particular errors when building from source.

If you have any problems, feel free to open an issue and myself or someone else might be able to help point you the right direction.
